eslint/config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
 	parser: 'babel-eslint',
 	extends: [
 		'wordpress',
+		'wordpress/es6',
```

Comment:
As discussed previously we might want to proceed with `es-next` to avoid confusion with all the recent versions of the JS language.